python/ray/tune/integration/keras.py

```diff
@@ -2,7 +2,7 @@
 from __future__ import division
 from __future__ import print_function
 
-import keras
+import tensorflow.keras as keras
```

Comment:
```suggestion
from tensorflow import tensorflow
```

Reply:
from tensorflow import tensorflow :) 

anyways, fixed to keras